package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "release-pr-action",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
```

Comment:
No this is expected for github actions. Either we have to push `node-modules` folder, or build a single file. Because it is going to be directly used by Github, and we cant have any intermediate steps to build the file.
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github